web/vtadmin/src/components/routes/workflows/WorkflowActions.tsx

```diff
@@ -44,11 +48,31 @@ interface CancelWorkflowOptions {
     keepRoutingRoules: boolean;
 }
 
+interface SwitchTrafficOptions {
+    enableReverseReplication: boolean;
+    force: boolean;
+    initializeTargetSequences: boolean;
```

Comment:
This is not a boolean anymore after: https://github.com/vitessio/vitess/pull/16860